source_sdk/user_cmd.h

```diff
@@ -12,6 +12,44 @@ struct user_cmd
     float sidemove;
     float upmove;
     int buttons;
+    unsigned char impulse;
```

Comment:
Not sure if the sizes of the data types are correct for this, got it from a very old thread on unknowncheats, but it works so far.

Reply:
Looks good I believe its missing `bool has_been_predicted`